lib/content-services/src/lib/document-list/components/file-auto-download/file-auto-download.component.html

```diff
@@ -0,0 +1,25 @@
+<div class="adf-non-responsive-dialog-container">
+    <div class="adf-dialog-header">
+        <h3 class="adf-mat-dialog-title">{{ 'ADF-DOCUMENT-LIST.FILE_AUTO_DOWNLOAD_DIALOG.HEADER' | translate }}</h3>
+    </div>
+    <div class="adf-dialog-body">
+        {{ 'ADF-DOCUMENT-LIST.FILE_AUTO_DOWNLOAD_DIALOG.LABEL' | translate }}
+    </div>
+    <div class="adf-dialog-footer">
+        <button
+            mat-button
+            id="cancelButton"
+            [attr.aria-label]="'ADF-DOCUMENT-LIST.FILE_AUTO_DOWNLOAD_DIALOG.ACTIONS.CANCEL' | translate"
+            (click)="onCancel()">
```

Comment:
you can just use `mat-dialog-close` attribute (check the docs), examples https://stackoverflow.com/questions/47592364/usage-of-mat-dialog-close

Reply:
Updated button HTML to use the mat-dialog-close directive. It passes the values of FileAduDownloadActionsEnum.CANCEL (for close button) and FileAduDownloadActionsEnum.DOWNLOAD (for download button)